edb/pgsql/metaschema.py

```diff
@@ -5254,29 +5254,94 @@ def _generate_sql_information_schema() -> List[dbops.View]:
         ),
     ]
 
+    pg_catalog_views = [
```

Comment:
Is the idea that we're only exposing the internals needed to get asyncpg working, for now?
(Document that, please)